kogito-quarkus-platform/pom.xml

```
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+         
+    <parent>
+        <groupId>org.kie.kogito</groupId>
+        <artifactId>kogito-build-parent</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+        <relativePath>../kogito-build-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>kogito-quarkus-parent-quarkus-platform-descriptor</artifactId>
```

What about this name?
```suggestion
    <artifactId>kogito-quarkus-platform-descriptor</artifactId>
```